plugins/generator-1.19.2/forge-1.19.2/procedures/world_loot_table_foreach.java.ftl

```diff
@@ -1,12 +1,12 @@
 <#include "mcelements.ftl">
 <#-- @formatter:off -->
 if (!world.isClientSide() && world.getServer() != null) {
-	BlockPos _bp = ${toBlockPos(input$x, input$y, input$z)};
+	BlockPos _bp${customBlockIndex} = ${toBlockPos(input$x, input$y, input$z)};

```

Comment:
Why is this addition needed? This block can not be nested anyway so as long as this field name is unique to this block, all is good.

Reply:
There are some other blocks using `_bp` for a technical variable name.

Reply:
Ok fair, that could be a problem, but just rename this one to _bpLotTblWorld and it is fine, no need for indexing

Reply:
So the idea is to create unique letter combinations for technical variables in all new blocks?

Reply:
When there is a possibility of collision. There can't be collision between two statement blocks without statement inputs so care/attention to this is not needed for those cases

Reply:
@Defeatomizer this part is still to be "fixed"

Reply:
_Yeah, I know..._